about.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name = "viewport" content="width=device-width, initial-scale=1">
+    <title>My Recipes</title>
+    <link rel="stylesheet" href="about.css">
+</head>
+
+<body>
+
+    <ul class="menubtn">
+        <li class="close_btn"><img src="images/close-btn.svg" alt="close-btn"></li>
+        <li class="menu-item"><a href="./index.html" class="nav-links">Home</a></li>
+        <li class="menu-item"><a href="./index.html#main" class="nav-links">Recipes</a></li>
+        <li class="menu-item"><a href="./about.html#head" class="nav-links">Contact</a></li>
+    </ul>
+    <nav class="navmenu">
+        <div class="nav-title">
+            <ul class="navlist">
+                <li><img src="images/twitter.png" alt="icon"></li>
+                <li><img src="images/facebook.png" alt="icon"></li>
+                <li><a href="#"> English</a></li>
+                <li><a href="./index.html"> MyPage</a></li>
+                <li><a href="#Contact"> Logout</a></li>
+            </ul>
+        </div>
+        <nav class="nav-bar">
+        <img src="images/christmas_2012_new_2857.jpg" alt="logo" class="logoicon">
+        <ul class="navitems">
+            <li><a href="./index.html#header" class="a-abt"> Home</a></li>
+            <li><a href="#"> Courses</a></li>
+            <li><a href="./index.html#main"> Recipes</a></li>
+            <li><a href="#Logo"> Favourites</a></li>
+            <li><a href="#head"> Contact</a></li>
+        </ul>
+        <ul class="linkbox">
+            <li><a href="#Contact"> MyRecipes</a></li>
+        </ul>
+        </nav>
+    </nav>
+    <header class="abt_heading" id="head">
+        <div class="header_bg">
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+            <div class="info-page">
+                <h2 class="abt-hello">"Hello! myFoodies"</h2>
+                <h1 class="abt-title">RECIPES TRIED BY IPTY</h1>
+                <div class="des-border">
+                <p class="abt-descrip">I would like to share recipes that I have tried making. I love making different kinds of food especially desserts and sharing with you all is amazing. </p>
+                </div>
+                <p class="abt-contact">Please contact me per Email for any further questions.</p>
+                <a href="#">mukadamiptisam@gmail.com</a> 
+            </div>
+        </div>           
+    </header>
+    <section class="main-about">
```

Comment:
- [Optional] Kindly consider using an indentation of two spaces in all your documents as this is a convention at microverse and also [best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).